instrumentation/opentelemetry-instrumentation-django/src/opentelemetry/instrumentation/django/middleware.py

```diff
@@ -295,7 +307,8 @@ def process_response(self, request, response):
             else:
                 activation.__exit__(None, None, None)
 
-        if self._environ_token in request.META.keys():
+        # if self._environ_token in request.META.keys():
+        if request.META.get(self._environ_token, None) is not None:
```

Comment:
How is this different from previous check?

Reply:
this will not fail even if the _environ_token is present in the request.META with None value.
Previously it will create an exception while calling detach() if the token is present with None value

Reply:
Yes